Simplified/Reader2/Internal/LibraryModule.swift

```diff
@@ -15,28 +15,67 @@ import R2Shared
 import R2Streamer
 import UIKit
 
-// TODO: SIMPLY-2656 Review "module" nomenclature
-
 /// The Library module handles the presentation of the bookshelf, and the publications' management.
 protocol LibraryModuleAPI {
-  var libraryService: LibraryService {get}
   
-  /// Loads the R2 DRM object for the given publication.
-  func loadDRM(for book: NYPLBook, completion: @escaping (CancellableResult<DRM?>) -> Void)
+  var delegate: LibraryModuleDelegate? { get }
+  
+  /// Root navigation controller containing the Library.
+  /// Can be used to present the library to the user.
+  var rootViewController: UINavigationController { get }
```

Comment:
I'm confused by the responsibilities of these various components: 
LibraryModule 
LibraryService
LibraryDelegate

I thought the service was the "backend" interface to R2 publications. The "Module" name never convinced me because it's vague (what is a module here?) but I kind of didn't care because it was such as a small class. The Delegate deals with UI I'm guessing?